docs/kyma/04-02-local-installation.md

```diff
@@ -37,7 +37,7 @@ Follow these instructions to install Kyma from a release or from sources:
      ```bash
      kyma provision minikube
      ```
-     >**NOTE:** The `provision` command uses the default Minikube VM driver installed for your operating system. For a list of supported VM drivers see [this document](https://kubernetes.io/docs/setup/minikube/#quickstart).
+     >**NOTE:** The `provision` command uses the default Minikube VM driver installed for your operating system. For more details, see [this document](https://kubernetes.io/docs/setup/minikube/).
```

Comment:
```suggestion
     >**NOTE:** The `provision` command uses the default Minikube VM driver installed for your operating system. For the list of supported VM drivers, see [this document](https://minikube.sigs.k8s.io/docs/start/).
```
The new link redirects to: https://kubernetes.io/docs/tasks/tools/.
Should we use this link instead? This note means to refer the reader specifically to supported drivers, not more details in general.

Reply:
I was thinking about adding the link you provided but wasn't sure whether I can use it or not 🤔 Since there's two of us now, I'll go with it, thanks! 😄